experimenter/experimenter/experiments/models.py

```diff
@@ -676,6 +676,23 @@ def results_ready(self):
             )
             return datetime.date.today() >= resultsReadyDate
 
+    @property
+    def has_displayable_results(self):
+        # True if self.results_data has weekly or overall results
+        if self.results_data and "v2" in self.results_data:
+            results_data = self.results_data["v2"]
+            for window in ["overall", "weekly"]:
+                if window in results_data:
+                    enrollments = results_data[window].get("enrollments", {}).get("all")
+                    if enrollments is not None:
+                        return True
```

Comment:
Oh awesome.  One thing I would've said here is having this logic live somewhere in the jetstream module and be callable from here, since it has to do with the details of the actual structure and content of the jetstream data, which in theory could be opaque to the experiment module, but this is a good step thank you 🙏

Reply:
Yea totally agree. I was planning to have a setter in the model that the Jetstream client code would call, but it wasn't working with the tests. I can revisit it separately though, and then this function can just be a regular getter.